bokehjs/src/coffee/models/annotations/label.coffee

```diff
@@ -143,6 +144,7 @@ class LabelView extends Renderer.View
         'font-size': "#{@_text_font_size[i]}"
         'font-family': "#{@mget('text_font')}"
         'background-color': "#{@visuals.background_fill.color_value()}"
+        'transform': "rotate(#{rad_to_deg(@_angle[i])}deg)"
```

Comment:
I'm a little confused - is angle always in radians?

I've been hunting around and I think the answer is yes. You can provide it in degrees python side (via AngleSpec), but then it comes down the pipe in radians? 

I just want to confirm this.